### DISCUSSION TITLE
Permission for creating a PR

### DISCUSSION THREAD

**Discussion by daniel-mezo** [webauthn4j/webauthn4j#748] [General]
Hi @ynojima, guys,

I'd like to create a small PR, but it seems I don't have permission to do so:
```
ERROR: Permission to webauthn4j/webauthn4j.git denied to daniel-mezo.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```
Can I be granted 'Open pull request' permission?

**Comment by ynojima** [comment]
If you would like to send a pull-request, please fork this repository to your account, push changes to your repository, and send a pull-request from your repository.
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request
